app/ore/permission/role/RoleTypes.scala

```diff
@@ -22,6 +22,7 @@ object RoleTypes extends Enumeration {
   val CommunityLeader = new  RoleType( 4, 59, classOf[GlobalRole], Default,  "Community Leader",   Amber)
   val Staff           = new  RoleType( 5, 3,  classOf[GlobalRole], Default,  "Sponge Staff",       Amber)
   val SpongeDev       = new  RoleType( 6, 41, classOf[GlobalRole], Default,  "Sponge Developer",   Green)
+  val OreDev          = new  RoleType(26, 65, classOf[GlobalRole], Default,  "Ore Developer",      Orange)
```

Comment:
this should be a group id of 66

Reply:
Is 65 being used somewhere?

Reply:
group id 65 is Issue_Manager on the forums

Reply:
Remind me to document this in the code.